src/msg/async/net_handler.cc

```diff
@@ -126,8 +129,6 @@ int NetHandler::generic_connect(const entity_addr_t& addr, bool nonblock)
     return -errno;
   }
 
-  set_socket_options(s);
-
```

Comment:
@yuyuyu101  you are right, i will change the title, and resubmit the pull request.